build.gradle.kts

```
@@ -1,27 +1,25 @@
 plugins {
     id("java-library")
-    id("com.github.johnrengelman.shadow")
-    id("biz.aQute.bnd.builder")
     id("maven-publish")
-    id("io.github.sgtsilvio.gradle.maven-central-publishing")
     id("signing")
-    id("com.github.hierynomus.license")
     id("pmd")
-    id("com.github.sgtsilvio.gradle.utf8")
-    id("com.github.sgtsilvio.gradle.metadata")
-    id("com.github.sgtsilvio.gradle.javadoc-links")
+    alias(libs.plugins.bnd)
+    alias(libs.plugins.javadoc.links)
+    alias(libs.plugins.license)
+    alias(libs.plugins.maven.central.publishing)
```

```suggestion
    alias(libs.plugins.mavenCentralPublishing)
```
some resources for naming version catalog entries: https://blog.gradle.org/best-practices-naming-version-catalog-entries, https://melix.github.io/blog/2021/03/version-catalogs-faq.html#_should_i_use_commons_lang3_as_an_alias_or_commonslang3